demo.env

```diff
@@ -44,26 +44,52 @@ ISLE_BUILD_TOOLS_BRANCH=7.x-1.13
 ## Islandora (DRUPAL)
 
 ### Islandora's User ID (UID) - for permission issues set this to your user's `id -u`
-ISLANDORA_UID=1000
+ISLANDORA_UID=10000
+ISLANDORA_GUID=10000
 
 ### Site name
-DRUPAL_SITE_NAME=ISLE.localdomain
+DRUPAL_SITE_NAME=isle-demo.localdomain
```

Comment:
So everyone using ISLE with the standard subdomain will need to change their /etc/hosts to have this new value

Reply:
Thank you for the careful analysis and I'm okay with these changes being reverted.